agent/result_parser.py

```diff
@@ -75,3 +74,12 @@ def get_list_from_string(scan_output_value: Union[str, List[str]]) -> List[str]:
         return [scan_output_value]
     else:
         return scan_output_value or []
+
+
+def format_str(value: str | List[str]) -> str:
+    """concat string list to match proto format
+
+    """
+    if isinstance(value, str):
+        return value
+    return reduce((lambda x, y: ' '.join([x, y])), value)
```

Comment:
Why this, reduce is the old way before list comprehensions, but it is not even needed.

return ' '.join(value)

is sufficient.